emitter/src/storage.rs

```diff
@@ -118,21 +118,21 @@ pub fn set_last_distro_time(e: &Env, last_distro: &u64) {
 /// Get whether the emitter has performed the drop distribution or not for the current backstop
 ///
 /// Returns true if the emitter has dropped
-pub fn get_drop_status(e: &Env) -> bool {
+pub fn get_drop_status(e: &Env, backstop: &Address) -> bool {
     e.storage()
         .instance()
-        .get(&EmitterDataKey::DropStatus)
-        .unwrap_optimized()
+        .get::<EmitterDataKey, bool>(&EmitterDataKey::Dropped(backstop.clone()))
+        .is_some()
```

Comment:
This feels like it could introduce a bug if changes are made down the line, as someone could store "false" here with the assumption the address can be dropped too.

Can we change `is_some()` to `unwrap_optimized()`?